packages/babel-core/src/index.ts

```diff
@@ -9,7 +9,14 @@ export const version = PACKAGE_JSON.version;
 export { default as File } from "./transformation/file/file.ts";
 export type { default as PluginPass } from "./transformation/plugin-pass.ts";
 export { default as buildExternalHelpers } from "./tools/build-external-helpers.ts";
-export { resolvePlugin, resolvePreset } from "./config/files/index.ts";
+
+import * as resolvers from "./config/files/index.ts";
+// For backwards-compatibility, we expose the resolvers
+// with the old API.
+export const resolvePlugin = (name: string, dirname: string) =>
+  resolvers.resolvePlugin(name, dirname, false).filepath;
+export const resolvePreset = (name: string, dirname: string) =>
+  resolvers.resolvePreset(name, dirname, false).filepath;
```

Comment:
Since they are public, can we add a few test cases for them? It can be done in another PR.